docs/Reducer.md

```diff
@@ -0,0 +1,28 @@
+## Reducer
+The reducer will handle all actions dispatched by the [action creators](ActionsCreators.md).
+
+### Configuration
+```
+import { reducer } from 'redux-clerk'
+
+const todos = reducer({
+  actionPrefix: 'TODOS'
+})
+
+export default todos
+```
+
+### Usage
+Once the reducer is configured, it will need added to the root reducer.
```

Comment:
This may be correct, you'll know better :) But I wanted to check if it was maybe better like this:
"..., it will need **to be** added to the root reducer."